index.html

```diff
@@ -8,7 +8,7 @@
     <style>
       @import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
     </style>
-    <title>Portafolio</title>
+    <title>Rafael Portafolio home page</title>
```

Comment:
Kindly update your page titles.

It adequately and briefly describes the content of the page. For example:
### Poor titles:
Microverse project | Your portfolio.
Portifolio project


### Better page titles:
Mark's portifolio
Mark full-stack webdeveloper